components/webui/client/src/pages/SearchPage/SearchControls/Presto/GuidedControls/index.module.css

```diff
@@ -1,17 +1,17 @@
 .gridContainer {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-template-columns: repeat(2, minmax(100px, 1fr));
     grid-auto-rows: 32px;
     gap: 5px;
 }
 
 .select {
```

Comment:
kinda unrelated to the changes in this PR - would it be cleaner to just write
```css
.select .from .where .order {
    grid-column: span 1;
    display: flex;
}
```
to avoid repeating the style definitions? if in the future they will have different styles, we can always split the block?